R/normalize.R

```diff
@@ -151,13 +151,16 @@ prep.step_normalize <- function(x, training, info = NULL, ...) {
 
 #' @export
 bake.step_normalize <- function(object, new_data, ...) {
-  check_new_data(names(object$means), object, new_data)
+  col_names <- names(object$means)
+  check_new_data(col_names, object, new_data)
 
-  for (column in names(object$means)) {
-    mean <- object$means[column]
-    sd <- object$sds[column]
-    new_data[[column]] <- (new_data[[column]] - mean) / sd
+  for (col_names in col_names) {
```

Comment:
Could we `col_name in col_names` here? The two uses of `col_names` have me 🤯

Reply:
It technically works, but I'm gonna fix it anyways :D 

``` r
x <- 1:5

for (x in x) {
  cat(x, "\n")
}
#> 1 
#> 2 
#> 3 
#> 4 
#> 5

x
#> [1] 5
```

<sup>Created on 2023-08-01 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>